docs/customize/configure.md

```diff
@@ -0,0 +1,621 @@
+# Configuration Variables
+
+This document provides a comprehensive list and explanation of the configuration variables available in the `invenio-app-ils` backend. These variables can be set in the configuration file(s) to customize the behavior of your Invenio ILS application.
+
+## REST Configurations
+
+### `ILS_RECORDS_REST_ENDPOINTS`
+
+REST endpoints for various record types.
+
+```python
+ILS_RECORDS_REST_ENDPOINTS = {
+    'books': {
+        'pid_type': 'bookid',
+        'path': '/books',
+        'default_media_type': 'application/json',
+    },
+    'members': {
+        'pid_type': 'memberid',
+        'path': '/members',
+        'default_media_type': 'application/json',
+    },
+}
+```
+
+### `ILS_MAILS_TEMPLATES`
+
+Paths to email templates used for various notifications.
+
+```python
+ILS_MAILS_TEMPLATES = {
+    'loan_due': 'invenio_app_ils/loans/email/loan_due.html',
+    'loan_overdue': 'invenio_app_ils/loans/email/loan_overdue.html',
+}
+```
+
+## Circulation Configuration
+
+### `CIRCULATION_LOAN_TRANSITIONS`
+
+A dictionary defining the possible transitions for loans.
+
+```python
+CIRCULATION_LOAN_TRANSITIONS = {
+    'CREATED': ['PENDING'],
+    'PENDING': ['ITEM_ON_LOAN', 'CANCELLED'],
+    'ITEM_ON_LOAN': ['RETURNED'],
+    'RETURNED': [],
+    'CANCELLED': [],
+}
+```
+
+For detailed documentation for circulation sub-module, refer to [invenio-circulation](https://invenio-circulation.readthedocs.io/en/latest/).
+
+## Debug Configuration
+
+### DEBUG (True/False)
+
+Enable or disable debugging mode.
+
+### DEBUG_TB_ENABLED (True/False)
+
+Enable or disable the debug toolbar.
+
+### DEBUG_TB_INTERCEPT_REDIRECTS (True/False)
+
+Enable or disable intercepting redirects in the debug toolbar.
+
+## Rate Limiting Configuration
+
+### RATELIMIT_STORAGE_URL
+
+Storage URL for the rate limiter.
+
+```python
+RATELIMIT_STORAGE_URL = "redis://localhost:6379/3"
```

Comment:
this is more "for website administrators" section of configuration, not features related one